### DISCUSSION TITLE
Move window to next/prev group

### DISCUSSION THREAD

**Discussion by mohaseeb** [qtile/qtile#3056] [Q&A] [answered]
From the documentation I can see that it's possible to move a `Window` to another `Group` using some identifier of it (i.e. `lazy.window.togroup("group_name")`). What I want to do is to be able to move a `Window` to the next or previous `Group` (e.g. `lazy.window.to_next_group()`, `lazy.window.to_prev_group()`).

Do you know how can I do that?

Thanks!

**Comment by elParaguayo** [comment, marked as answer]
You should be able to do something like this:
``` python
@lazy.window.function
def window_to_next_group(window):
    index = window.qtile.groups.index(window.group)
    index = (index + 1) % len(window.qtile.groups)
    window.cmd_togroup(window.qtile.groups[index].name)

# Your keybinding would just be to `window_to_next_group`
```

Untested and I wouldn't be surprised if I've missed something blindingly obvious. I usually do.

**Reply by mohaseeb** [reply]
Thanks @elParaguayo; it worked flawlessly :+1:

**Reply by ghost** [reply]
how should be included the function and the keybinding in the conf file?
